src/css/custom.css

```diff
@@ -322,23 +322,28 @@ aside.theme-doc-sidebar-container {
   margin-top: 0;
 }
 
+.algolia-docsearch-suggestion--category-header {
+  background-color: var(--ifm-color-gray-300) !important;
```

Comment:
gray should not be used in the name you can name it, base-primary or base-secondary

Reply:
Hey, these are pre-defined docusauras variables.
https://github.com/facebook/docusaurus/issues/3955#issuecomment-1521944593